app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from os import makedirs, remove, path
+
+
+def move_file(command: str) -> None:
+    sep_command = command.split()
+
+    if len(sep_command) != 3:
+        print("Incorrect command!")
+
+    file_in = sep_command[1]
+    file_out = sep_command[2]
```

Comment:
First of all. These are not files.
```suggestion
    cmd, source_path, destination_path = sep_command
```